mlir/unittests/programs/quantum_computation_programs.cpp

```
@@ -88,458 +98,550 @@ void repeatedResetAfterSingleOp(QuantumComputation& comp) {
   comp.i(0);
+  comp.measureAll(true, false);
 }
 
 void singleControlledIdentity(QuantumComputation& comp) {
   comp.addQubitRegister(2, "q");
-  comp.ci(1, 0);
+  comp.ci(0, 1);
+  comp.measureAll(true, false);
 }
 
 void multipleControlledIdentity(QuantumComputation& comp) {
```

Cppcheck (2.21.0)

[style] The function 'multipleControlledIdentity' is never used.

(unusedFunction)